Samples.xcodeproj/project.pbxproj

```diff
@@ -1482,9 +1508,9 @@
 				D752D9462A3A6F80003EB25E /* MonitorChangesToMapLoadStatusView.swift in Sources */,
 				E0082217287755AC002AD138 /* View+Sheet.swift in Sources */,
 				00181B462846AD7100654571 /* View+Alert.swift in Sources */,
+				D7ABA2FF2A32881C0021822B /* ShowViewshedFromGeoelementInSceneView.swift in Sources */,
 				E0FE32E728747778002C6ACA /* BrowseBuildingFloorsView.swift in Sources */,
 				D752D95F2A3BCE06003EB25E /* DisplayMapFromPortalItemView.swift in Sources */,
-				1CAB8D072A3CCBD8002AA649 /* SetVisibilityOfSubtypeFeatureLayerView.Views.swift in Sources */,
```

Comment:
This seems out of place in this PR. If this is needed, ideally it would be done in a separate PR (so that it gets done ASAP, since it affects more than this PR) and then this branch would be updated/rebased to include that change, if needed.

Reply:
This was already fixed in [#223](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/223) as was discessed here [#223 (comment)]( https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/223#discussion_r1256449340). Do I still need to move this change out of this branch?

Reply:
I'll defer to @yo1995 on this one since he's already talked with you about it.

Reply:
I'm not sure why it is still shown as a diff - it is not on [`v.next`](https://github.com/Esri/arcgis-maps-sdk-swift-samples/blob/v.next/Samples.xcodeproj/project.pbxproj#L1487)